docker/supabase/docker-compose.yml

```diff
@@ -32,9 +32,25 @@ services:
       # https://github.com/supabase/cli/issues/14
       KONG_DNS_ORDER: LAST,A,CNAME
       KONG_PLUGINS: request-transformer,cors,key-auth,acl
+      JWT_ISSUER: ${JWT_ISSUER}
+      JWT_AUDIENCE: ${JWT_AUDIENCE}
+      SECRET: ${SECRET}
     volumes:
       - ./supabase/volumes/api/kong.yml:/var/lib/kong/kong.yml
+    depends_on:
+      - traefik
+    labels:
+      # Enable Traefik
+      - 'traefik.enable=true'
+      # Traefik routing for the kong service at /v1/supabase
+      - 'traefik.http.routers.kong.rule=PathPrefix(`/${API_VERSION}/supabase`)'
+      # Specify the kong service port
+      - 'traefik.http.services.kong.loadbalancer.server.port=8000'
 
+      # Add middleware to this route to strip the /v1/image prefix
+      - 'traefik.http.middlewares.strip_kong_prefix.stripprefix.prefixes=/${API_VERSION}/supabase'
+      - 'traefik.http.middlewares.strip_kong_prefix.stripprefix.forceSlash=true'
+      - 'traefik.http.routers.image.middlewares=strip_kong_prefix'
```

Comment:
copy/paste error here: `routers.image`.  Should be `routers.kong`.

Reply:
This is working